cmd/pmm-managed-init/main.go

```diff
@@ -17,15 +17,25 @@
 package main
 
 import (
-	"flag"
-	"log"
 	"os"
+
+	"github.com/sirupsen/logrus"
+
+	"github.com/percona/pmm-managed/utils/validators"
 )
 
 func main() {
-	flag.Parse()
+	l := logrus.WithField("component", "pmm-managed-init")
```

Comment:
```
time="2020-02-21T06:37:09Z" level=warning msg="unknown environment variable \"PWD=/opt\"" component=pmm-managed-init
time="2020-02-21T06:37:09Z" level=warning msg="unknown environment variable \"SHLVL=1\"" component=pmm-managed-init
time="2020-02-21T06:37:09Z" level=warning msg="unknown environment variable \"_=/usr/sbin/pmm-managed-init\"" component=pmm-managed-init
time="2020-02-21T06:37:09Z" level=error msg="environment variable \"DATA_RETENTION=25h\" should be a natural number of 24h0m0s" component=pmm-managed-init
```

That's not user-friendly. Copy and tweak formatter from main.go, and do not use fields or caller info.